plugin.video.nhkworldtv/addon.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.nhkworldtv" name="NHK World TV" version="1.0.0+matrix.1" provider-name="sbroenne">
+    <requires>
+        <import addon="xbmc.python" version="3.0.0"/>
+        <import addon="script.module.certifi" version="2019.9.11"/>
+        <import addon="script.module.requests" version="2.22.0"/>
+        <import addon="script.module.requests-cache" version="0.4.13"/>
+        <import addon="script.module.routing" version="0.2.3"/>
+        <import addon="script.module.future" version="0.17.1"/>
+        <import addon="script.module.kodi-six" version="0.1.3"/>
+        <import addon="script.module.pytz" version="2014.2"/>
+        <import addon="script.module.tzlocal" version="2.0.0"/>
+    </requires>
+
+    <extension point="xbmc.python.pluginsource" library="main.py">
+        <provides>video</provides>
+    </extension>
+
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">Watch NHK World Japan in Full HD - live stream, on-demand and news programs</summary>
+        <description lang="en_GB">
+Watch NHK World Japan in Full HD[CR]
+NHK provides streams via Akamai in 720p and 1080p - the plug-in defaults to 1080p where available.[CR]
+If you encounter buffering issues, you can fall-back to 720p by changing the 'Use 720p instead of 1080p' addon-setting.[CR]
+Tested on Kodi 18.6+ and 19.[CR]
+        </description>
+        <disclaimer lang="en_GB">This plugin is a fan project and not related in any way to NHK!</disclaimer>
+        <platform>all</platform>
+        <license>GPL-3.0</license>
```

Comment:
This should be "GPL-3.0-only" or "GPL-3.0-or-later" depending on how you license your plugin (valid spdx identifiers)